src/components/scenes/CoinRankingDetailsScene.tsx

```diff
@@ -104,7 +105,7 @@ const CoinRankingDetailsSceneComponent = (props: Props) => {
         return data == null ? 'N/A' : toPercentString(Number(data) / 100)
       case 'price':
       case 'priceChange24h':
-        return formatFiatString({ fiatAmount: baseString })
+        return `${formatFiatString({ fiatAmount: baseString })} ${USD_FIAT.replace('iso:', '')}`
```

Comment:
Maybe there's a good reason but I couldn't figure out why this feature is hardcoded to USD. I added a small fixup commit that uses the user's default fiat instead of USD for both components. If you use it the commit message should be changed before merging. 

If there is a good reason to keep USD only feel free to drop the fixup and merge since it would be approved anyway.

Reply:
That's legitimate - it used to be hardcoded to USD before because we were sourcing an API that only reliably had US quotes.

Reply:
Avoiding scope creep since this PR is meant to be a fix for the rank and USD display. Also the proposed change would break the feature for fiat types not supported by the API.

Tracked this new idea in task: https://app.asana.com/0/1200382638405084/1203963276097127/f